File: Models/Db/Word.cs
using System;
using System.Collections.Generic;

namespace PostponedWords.Models.Db
{
    public partial class Word
    {
        public int Id { get; set; }
        public string WordText { get; set; }
        public string Meaning { get; set; }
        public string Example { get; set; }
    }
}

File: ClientApp/app/models/word.ts
export class Word
{
	word: string;
	meaning: string;
	example: string;
	id: number;
	constructor(word?: string, meaning?: string, example?: string, id?: number)
	{
		this.word = word ? word : "";
		this.meaning = meaning ? meaning : "";
		this.example = example ? example : "";
		this.id = id ? id : -1;
	}
}